variables.tf

```diff
@@ -123,3 +123,9 @@ variable "disable_agent" {
   description = "User defined status of the agent. Set to `true` to disable the agent. By default the agent state will be enabled."
   default     = false
 }
+
+variable "schematics_policies" {
+  description = "Schematics agent policies to create"
+  type        = any
```

Comment:
we should not use `any` when we know the schema. Please add the full schema here. We only use `any` if we do not know the schema. 

I'm also wondering should this not be a list(object) type? A list of policies? The default value would indicate its a map. You probably also need to set this to `nullable = false`